src/renderer/services/ethereum/common.ts

```diff
@@ -46,11 +46,11 @@ const clientState$ = Rx.combineLatest([keystoreService.keystore$, ethereumNetwor
           getPhrase(keystore),
           O.chain((phrase) => {
             try {
-              const blockchairUrl = network === 'testnet' ? BLOCKCHAIR_TESTNET : BLOCKCHAIR_MAINNET
+              const ethplorerUrl = network === 'testnet' ? ETHPLORER_TESTNET : ETHPLORER_MAINNET
               const client = new Client({
                 network,
-                blockchairUrl: blockchairUrl,
-                blockchairNodeApiKey: BLOCKCHAIR_API_KEY,
+                ethplorerUrl,
```

Comment:
`ethplorerUrl` is not a string anymore, it's an object right now ([check `ClientUrl`](https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-ethereum/src/types/client-types.ts#L11-L14))